webcompat/static/js/lib/css-fixme-ui.js

```diff
@@ -0,0 +1,26 @@
+/* global doTheBigStyleFixing */
+addEventListener('load', function() {
+
+  function do_fixup() {
+    var csscode = document.getElementsByTagName('textarea')[0].value;
+    var pre = document.getElementById('fixedcss');
+    var compact = document.getElementsByName('compact')[0].checked;
+    doTheBigStyleFixing(csscode, pre, compact);
+  }
+
+  document.getElementById('btn_do_fixup').addEventListener('click', do_fixup, false);
+
+  if (document.getElementsByTagName('textarea')[0].value) {
+    do_fixup();
+  }
+
+  document.getElementById('fixedcss')[0].addEventListener('dblclick', function(e) {
```

Comment:
Is this a typo? `document.getElementById('fixedcss')[0]` will be undefined, I think.